example/build.gradle

```diff
@@ -47,4 +48,25 @@ dependencies {
     reactor_3_3_x 'io.projectreactor:reactor-core:3.3.0.BUILD-SNAPSHOT', {
         changing = true
     }
+
+    jmh project(path: ":agent", configuration: 'shadow')
+    //Uncomment for benchmark execution by Idea plugin
+//    jmh "org.openjdk.jmh:jmh-generator-annprocess:1.21"
+}
+
+jmh {
+    if (project.hasProperty("jmh.simpleTravisMode")) { //temporary for JHM sanity testing in Travis
```

Comment:
leftover?

Reply:
No, it's on purpose in the first stage - due to aforementioned local issues with C compilation.